### ISSUE TITLE
Change newsletter sign-up

### ISSUE THREAD

**Issue by MyfanwyNixon** [mysociety/internal#31]
Breaking this out from #25 into its own ticket: 

> Alter the sign-up form so that people self-select their segment at point of subscription.

See also #26 - this will have a knock-on effect everywhere. 

Do we want a single form that will automatically place people in the right segment, when they're signing up from obvious places such as the FMS for Councils page? I think the ideal would be:
- the form invites you to sign up for the Better Cities newsletter
- you are asked to tick boxes to indicate if you're also interested in the other areas (FOI for journalists and campaigning - and maybe 'general mySociety news')

**Comment by MyfanwyNixon** [comment]
@abimysoc can we add this to the next internal prioritisation meeting please.

**Comment by MyfanwyNixon** [comment]
@jacksonj04 I'd be really grateful if you could also help me with thinking through the newsletter to existing users which invites them to sign up for a specific segment. 

What would be the user experience? - they check a box or click a button to pick the newsletter they want and then what? Can we recognise who they are? :/

**Comment by MyfanwyNixon** [comment]
@jacksonj04 to add, ooh I think this might be one answer: http://kb.mailchimp.com/campaigns/design/send-a-simple-poll-or-survey-to-subscribers
> After your subscribers begin to vote, you can create segments based on the results. On polls, you can segment by whether subscribers have responded yet. For surveys, you can segment by specific responses or by whether the survey has been completed.

**Comment by MyfanwyNixon** [comment]
Quick note to self - we'll also need to look at the 'thanks for subscribing' trigger emails once this is in place.

**Comment by MyfanwyNixon** [comment]
The suggested newsletter text is drafted as 'choose your segment' in Mailchimp. https://us9.admin.mailchimp.com/campaigns/

**Comment by MyfanwyNixon** [comment]
@jacksonj04 Mornin'
I dropped mailchimp a quick line about this since we get priority support, being paid customers. Here's what they suggest:

> Create 4 Groups within your list > send an email to the entire list > within the email ask that the subscribers opt in to a Group(s) that aligns with their interests. This can be done by including an 'update profile' link in the email sent to the list. 

> Ultimately, this would achieve our objective here as it would allow for your subscribers to do the bulk of the work by self-identifying which Group(s) they would like to join. An added benefit would be that by creating Groups on the list these would be visible on your signup form as well. As such, as subscribers join the list in the first place they could effectively self-identify themselves. 

> That said, here's a look at how to Create a New List Group: http://eepurl.com/cdy2K1 as well as a look at how to Add an Update Profile Link: http://eepurl.com/b2Q7_r

**Comment by jacksonj04** [comment]
Current status:

* [x] Create groups in MailChimp (https://us9.admin.mailchimp.com/lists/dashboard/groups?id=425649)
* [x] Make groups visible in profile update form (use the `*|UPDATE_PROFILE|*` merge tag in the email) and native new subscriber form (http://eepurl.com/3BHlj)
* [x] Change Wordpress ACF to allow for groups and group options
* [ ] Change Wordpress theme to make use of new options (https://github.com/mysociety/orgsites/pull/933)
* [ ] :shipit:
